frontend/src/metabase/visualizations/components/TableSimple.jsx

```diff
@@ -137,6 +140,13 @@ export default class TableSimple extends Component {
       }
     }
 
+    let paginateMessage;
+    if (!limit || limit >= 2000) {
```

Comment:
Minor thing, but I'd prefer `limit === undefined` when zero is a valid option. Unlikely to see that here, but it helps avoid any quirks around falsey zeros.